src/Component/Symfony/Routing/Loader/ResourceLoader.php

```diff
@@ -0,0 +1,203 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Component\Resource\Symfony\Routing\Loader;
+
+use Gedmo\Sluggable\Util\Urlizer;
```

Comment:
If nature of PR allows it, let's switch to `use Behat\Transliterator\Transliterator;` directly

![image](https://github.com/Sylius/SyliusResourceBundle/assets/870747/bbb81cbb-2165-4169-a2b1-574db06665ca)

https://github.com/doctrine-extensions/DoctrineExtensions/blob/main/src/Sluggable/Util/Urlizer.php

Reply:
Extracted to https://github.com/Sylius/SyliusResourceBundle/issues/753